File: modules/076-skills-management-system/tools2/src/commands/aihelp.ts
import * as fs from 'fs';
import * as path from 'path';

export function runAIHelp(): void {
  // Try to find SKILL-CLI.md relative to the script
  const candidates = [
    path.join(__dirname, '..', '..', 'SKILL-CLI.md'),  // from dist/commands/
    path.join(__dirname, '..', 'SKILL-CLI.md'),
  ];

  for (const p of candidates) {
    if (fs.existsSync(p)) {
      console.log(fs.readFileSync(p, 'utf8'));
      return;
    }
  }

  console.error('Error: SKILL-CLI.md not found.');
  console.error('See https://github.com/codenjoyme/apm-lite/blob/master/SKILL-CLI.md');
  console.error('Or use `skills help` for basic usage.');
  process.exit(1);
}

File: modules/076-skills-management-system/tools/scripts/cmd/aihelp.go
package cmd

import (
	"fmt"
	"os"
	"path/filepath"
)

// RunAIHelp handles the `skills ai-help` command — prints the SKILL-CLI.md reference.
func RunAIHelp(_ []string) {
	// Try to find SKILL-CLI.md relative to the executable
	exePath, err := os.Executable()
	if err == nil {
		skillCliPath := filepath.Join(filepath.Dir(exePath), "..", "SKILL-CLI.md")
		data, err := os.ReadFile(skillCliPath)
		if err == nil {
			fmt.Print(string(data))
			return
		}
	}
	fmt.Fprintln(os.Stderr, "Error: SKILL-CLI.md not found.")
	fmt.Fprintln(os.Stderr, "See https://github.com/codenjoyme/apm-lite/blob/master/SKILL-CLI.md")
	fmt.Fprintln(os.Stderr, "Or use `skills help` for basic usage.")
	os.Exit(1)
}
